src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -216,12 +216,21 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
 
   </script>
 
-  <div id="subscribeButton">
-    <button type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false"
-       autocomplete="off">
-      Subscribe
-    </button>
-  </div>
+
+   <button id="subscriptionButton" class="btn btn-primary"  onclick="myFunction()"> subscribe </button>
```

Comment:
Let's rename the function to something more expressive. Maybe "`subscribeToConversation`" or something?